opentelemetry/src/main/scala/zio/telemetry/opentelemetry/tracing/Tracing.scala

```diff
@@ -717,10 +717,13 @@ object Tracing {
             val errorStatus =
               cause.failureOption
                 .flatMap(errorMapper.body.lift)
-                .getOrElse(StatusCode.UNSET)
+                .getOrElse(StatusCode.ERROR)
 
             for {
-              _      <- ZIO.succeed(span.setStatus(errorStatus, cause.prettyPrint))
+              _      <- ZIO.ifZIO(ZIO.succeed(errorStatus == StatusCode.ERROR))(
+                          onTrue = ZIO.succeed(span.setStatus(errorStatus, cause.prettyPrint)),
+                          onFalse = ZIO.succeed(span.setStatus(errorStatus))
+                        )
```

Comment:
I suggest changing it to a simple if/else statement like this:
```
              _      <- if (errorStatus == StatusCode.ERROR)
                          ZIO.succeed(span.setStatus(errorStatus, cause.prettyPrint))
                        else
                          ZIO.succeed(span.setStatus(errorStatus))
```

Reply:
Ok, fixed